sdk/rust/oak/src/logger/mod.rs

```diff
@@ -44,7 +44,7 @@ impl Log for OakChannelLogger {
             level: map_level(record.level()) as i32,
             message: format!("{}", record.args()),
         };
-        match self.log_sender.send(&log_msg) {
+        match self.log_sender.send_with_downgrade(&log_msg) {
```

Comment:
This does not feel right, why is every log statement performing a downgrade? This means that if the node has sufficient privilege to declassify to public, it will be able to actually log, which is probably not the intention.

To clarify: I expect that non-public nodes will be able to log in debug (where the log node is top secret) but not in production (where the log node is public).

Reply:
Removed the downgrade privilege and stopped panicking when the send is blocked by IFC.